cmd/heartbeats/main.go

```diff
@@ -50,39 +46,57 @@ func init() {
 
 func main() {
 	flag.Parse()
+	var period time.Duration
+	if p, err := strconv.Atoi(periodStr); err != nil {
```

Comment:
What about using `flag.IntVar` to move this conversion into the flag library?

Reply:
it can't because of the array of strings in the Container Source